frontend/package.json

```diff
@@ -32,5 +33,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "https://dry-wildwood-77221.herokuapp.com"
+  "proxy": "http://localhost:8080"
```

Comment:
To trzeba cofnąć

Reply:
Tak, zmieniałem u siebie, musiałem zapomnieć odznaczyć przy commitowaniu plik. Przy ostatnim commicie będzie zmienione.